### DISCUSSION TITLE
CommonMark 0.30 is out

### DISCUSSION THREAD

**Discussion by davidsneighbour** [yuin/goldmark#233] [General]
I hate to be one of the "is it done yet?" people, but I just started to dig deeper into the "commonmark compliance" and what it means for specific markdown features and found, that there is very young new version 0.30 specification out. I could not find any issue or discussion here about that and wonder, if you see any problem with the changes they propose? It's mostly wording and very specific instructions on how to escape special characters. 

https://spec.commonmark.org/
https://spec.commonmark.org/0.30/changes.html

**Comment by yuin** [comment]
Thank you for letting me know about 0.30.
I've made a v1.4.0 release of the goldmark that supports commonmark 0.30.
